File: src/test/java/com/github/vbauer/avconv4java/option/AVSubtitleOptionsTest.java
package com.github.vbauer.avconv4java.option;

import org.testng.annotations.Test;
import org.testng.collections.Lists;

import java.util.List;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.emptyIterable;
import static org.hamcrest.Matchers.not;

/**
 * @author Vladislav Bauer
 */

public class AVSubtitleOptionsTest {

    @Test
    public void testSmokeAVSubtitleOptions() {
        final List<String> options = AVSubtitleOptions.create()
            .disableRecording()
            .subtitleCodec("codecName")
            .builders(Lists.newArrayList())
            .builders()
            .flags(Lists.newArrayList())
            .flags()
            .build();

        assertThat(options, not(emptyIterable()));
    }

}

File: src/test/kotlin/com/github/vbauer/avconv4java/option/AVSubtitleOptionsTest.kt
package com.github.vbauer.avconv4java.option

import org.hamcrest.MatcherAssert.assertThat
import org.hamcrest.Matchers.emptyIterable
import org.hamcrest.Matchers.not
import org.testng.annotations.Test
import org.testng.collections.Lists

/**
 * @author Vladislav Bauer
 */

class AVSubtitleOptionsTest {

    @Test
    fun testSmokeAVSubtitleOptions() {
        val options = AVSubtitleOptions.create()
                .disableRecording()
                .subtitleCodec("codecName")
                .builders(Lists.newArrayList())
                .builders()
                .flags(Lists.newArrayList())
                .flags()
                .build()

        assertThat(options, not(emptyIterable()))
    }

}
